docs/syntax.html

```diff
@@ -315,7 +315,7 @@ <h1 id="locale-and-keypath-syntax">Locale and KeyPath Syntax</h1><p>You can use
   <span class="hljs-comment">&lt;!-- nested --&gt;</span>
   <span class="hljs-tag">&lt;<span class="hljs-name">p</span>&gt;</span>{{ $t(&apos;nested.message1&apos;) }}<span class="hljs-tag">&lt;/<span class="hljs-name">p</span>&gt;</span>
   <span class="hljs-comment">&lt;!-- array --&gt;</span>
-  <span class="hljs-tag">&lt;<span class="hljs-name">p</span>&gt;</span>{{ $t(&apos;errros[0]&apos;) }}<span class="hljs-tag">&lt;/<span class="hljs-name">p</span>&gt;</span>
+  <span class="hljs-tag">&lt;<span class="hljs-name">p</span>&gt;</span>{{ $t(&apos;errors[0]&apos;) }}<span class="hljs-tag">&lt;/<span class="hljs-name">p</span>&gt;</span>
```

Comment:
`docs/sytax.html` is generated from [gitbook/syntax](https://github.com/kazupon/vue-i18n/blob/dev/gitbook/syntax.md).
Can you update the gitbook doc file please?